parsers/rhea/rhea.go

```diff
@@ -0,0 +1,791 @@
+package rhea
+
+import (
+	"compress/gzip"
+	"database/sql"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+)
+
+/******************************************************************************
+
+Lower level structs
+
+These structs operate at the lowest level to parse the Rhea Rdf database dump
+into structs that Golang can understand. Getting all of Rhea from Rdf->Golang
+is quite verbose, and so most of the time you should not use these structs unless
+you know what you are doing and it cannot be accomplished with higher level structs.
+
+******************************************************************************/
+
+// Rdf is the RDF XML representation of the Rhea database.
+type Rdf struct {
+	XMLName      xml.Name      `xml:"RDF"`
+	Descriptions []Description `xml:"Description"`
+}
+
+// Description is the XML representation of an object in the Rhea database.
+type Description struct {
+	// Reaction
+	XMLName              xml.Name             `xml:"Description"`
+	About                string               `xml:"about,attr"`
+	ID                   int                  `xml:"id"`
+	Accession            string               `xml:"accession"`
+	Equation             string               `xml:"equation"`
+	HTMLEquation         string               `xml:"htmlEquation"`
+	IsChemicallyBalanced bool                 `xml:"isChemicallyBalanced"`
+	IsTransport          bool                 `xml:"isTransport"`
+	Citations            []Citation           `xml:"citation"`
+	Substrates           []Substrate          `xml:"substrates"`
+	Products             []Product            `xml:"products"`
+	SubstrateOrProducts  []SubstrateOrProduct `xml:"substratesOrProducts"`
+	Subclass             []Subclass           `xml:"subClassOf"`
+	Comment              string               `xml:"comment"`
+	EC                   EC                   `xml:"ec"`
+	Status               Status               `xml:"status"`
+	Compound             CompoundXML          `xml:"compound"`
+
+	// ReactionSide / Reaction Participant
+	BidirectionalReactions []BidirectionalReaction `xml:"bidirectionalReaction"`
+	DirectionalReactions   []DirectionalReaction   `xml:"directionalReaction"`
+	Side                   Side                    `xml:"side"`
+	SeeAlsos               SeeAlso                 `xml:"seeAlso"`
+	TransformableTo        TransformableTo         `xml:"transformableTo"`
+	CuratedOrder           int                     `xml:"curatedOrder"`
+	Contains               Contains                `xml:"contains"`
+
+	// ContainsX contains all other name-attribute pairs, with names like "contains1" in mind
+	ContainsX []ContainsX `xml:",any"`
+
+	// Small Molecule tags
+	Name     string   `xml:"name"`
+	HTMLName string   `xml:"htmlName"`
+	Formula  string   `xml:"formula"`
+	Charge   string   `xml:"charge"`
+	Chebi    ChebiXML `xml:"chebi"`
```

Comment:
```suggestion
	ChEBI    ChEBIXML `xml:"chebi"`
```
maybe follow the correct capitalization here and in the rest of the `*ChEBI*` variable names?

https://en.wikipedia.org/wiki/ChEBI

Reply:
Good catch!